spec/controllers/env_settings_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe EnvSettingsController do
+  context 'GET index' do
+    before do
+
```

Comment:
please use more accurate example data. this makes it more understandable :)

e.g. fake-link -> https://help.example.com